node/pkg/txverifier/utils.go

```diff
@@ -96,3 +98,10 @@ func denormalize(
 
 	return denormalizedAmount
 }
+
+// SupportedChains returns a slice of Wormhole Chain IDs that have a Transfer Verifier implementation.
+func SupportedChains() []vaa.ChainID {
```

Comment:
So for now, you are only supporting Ethereum, not any of the other EVMs (not even Sepolia or Holeksy)?

Reply:
Yes, but adding the testnets should be OK.

Reply:
Testnets added in latest commit.